src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -373,4 +373,17 @@ function MdProgressCircularDirective($window, $mdProgressCircular, $mdTheming,
     return $mdProgressCircular.strokeWidth / 100 * diameter;
   }
 
+  /**
+   * Return circumference of the spinner
+   *
+   * @param {number} diameter Diameter of the container.
+   * @param {number} strokeWidth Stroke width to be used when drawing circle
+   * @param {number} fraction Fraction of circumference
```

Comment:
It would be helpful to add some additional detail here. Perhaps how a developer would get this value.

Reply:
I refactored the code and the comments to make it more clear.